share/smack/frontend.py

```diff
@@ -250,6 +250,8 @@ def default_build_libs(args):
   if args.float:
     libs += ['math.c']
     libs += ['fenv.c']
+    if args.bit_precise:
+      libs += ['intrinsics.c']
```

Comment:
Why are we not adding `intrinsics.c` to `libs` always?

Reply:
I think we can do it in the coming PR to refactor LLVM intrinsic handling.